lib/default.nix

```diff
@@ -226,7 +226,13 @@ let
           module = import modulePath;
         in
         if expectsPublisherArgs module then
-          lib.setDefaultModuleLocation modulePath (module publisherArgs)
+          {
+            _file = modulePath;
+            imports = [
+              (module publisherArgs)
+              perSystemModule
```

Comment:
will this introduce the proper `perSystem` for nixos and darwin modules?

Reply:
I do think so? I'll test it a bit further on both platforms before undrafting. Just updated priorities